package-parser/package_parser/processing/migration/_differ.py

```diff
@@ -0,0 +1,33 @@
+from abc import ABC, abstractmethod
+
+from package_parser.processing.api.model import Class, Function, Parameter, Result
+
+
+class AbstractDiffer(ABC):
+    @abstractmethod
+    def compute_attribute_similarity(
+        self,
+        attributes_a: str,
```

Comment:
Don't we have a class to represent attributes? If not, that would be helpful since we need to store their name and, if possible, their type.